src/patterns/question-pages/index.md.njk

```diff
@@ -22,7 +22,12 @@ You should make sure you know why you’re asking every question and only ask us
 
 To help you work out what to ask, you can carry out a [question protocol](https://www.uxmatters.com/mt/archives/2010/06/the-question-protocol-how-to-make-sure-every-form-field-is-necessary.php).
 
-If you ask for optional information, mark the labels of optional fields with ‘(optional)’. Never mark mandatory fields with asterisks.
+If you ask for optional information:
+
+- in most contexts, add ‘(optional)’ to the labels of optional fields```
+- for [radios](https://design-system.service.gov.uk/components/radios/) and [checkboxes](https://design-system.service.gov.uk/components/checkboxes/), add ‘(optional)’ to the legend
```

Comment:
```suggestion
- for [radios](/components/radios/) and [checkboxes](/components/checkboxes/), add ‘(optional)’ to the legend
```
Sorry I didn't spot this before – should change this to use relative links rather than URLs

Reply:
My bad, I often forget to do that - thanks @christopherthomasdesign!